dali/ft/filecopy.cpp

```diff
@@ -3136,12 +3131,17 @@ void FileSprayer::updateTargetProperties()
                      ((stricmp(aname,FArecordCount)==0)&&!gotrc) ||
                      ((stricmp(aname,"@blockCompressed")==0)&&copyCompressed) ||
                      ((stricmp(aname,"@rowCompressed")==0)&&copyCompressed)||
-                     (stricmp(aname,"@local")==0)
+                     (stricmp(aname,"@local")==0)||
+                     (stricmp(aname,"@checkSum")==0)||
```

Comment:
Actually the spray/copy doesn't calculate CRC if the spray/copy target compressed or copy compressed source file to compressed target, based on  this function:

bool FileSprayer::calcCRC()
{
    return options->getPropBool(ANcrc, true) && !compressOutput && !copyCompressed; 
}